prog/weaveutil/main.go

```diff
@@ -28,10 +31,24 @@ func init() {
 		"del-iface":              delIface,
 		"setup-iface":            setupIface,
 		"list-netdevs":           listNetDevs,
+		"cni-net":                cniNet,
+		"cni-ipam":               cniIPAM,
 	}
 }
 
 func main() {
+	weavenet.WeaveUtilCmd = "/proc/self/exe" // force re-exec of this binary
```

Comment:
This trick does not work because `WithNetNS` called inside `weaveutil` will exec `nsenter --net=<..> /proc/self/exe <..>`, where `/proc/self/exe` links to `nsenter`.